lib/time_turner/generator.rb

```diff
@@ -0,0 +1,30 @@
+require 'active_support/core_ext/integer'
+
+module TimeTurner
+  class Generator
+    def initialize(seed)
+      @seed = seed
+      @rng = Random.new(seed)
```

Comment:
```suggestion
      @rng = Random.new(seed)
```
Il faut éviter de tronquer les noms de variables comme ça. Perso j'arrive même pas à voir ce que rng signifie. j'aurais dit `range` mais ça me semble pas être le cas. Juste `@random` aurait suffi ici par exemple